src/pip/_internal/cli/cmdoptions.py

```diff
@@ -476,11 +476,12 @@ def no_binary():
         callback=_handle_no_binary, type="str",
         default=format_control,
         help="Do not use binary packages. Can be supplied multiple times, and "
-             "each time adds to the existing value. Accepts either :all: to "
-             "disable all binary packages, :none: to empty the set, or one or "
-             "more package names with commas between them (no colons). Note "
-             "that some packages are tricky to compile and may fail to "
-             "install when this option is used on them.",
+             "each time adds to the existing value. Accepts either \":all:\" "
+             "to disable all binary packages, \":none:\" to empty the set "
+             "(notice the colons), or one or more package names with commas "
+             "between them (no colons). Note that some packages are tricky to "
+             "compile and may fail to install when this option is used on "
+             "them.",
```

Comment:
I reordered the words to resolve the "E501 line too long" lint error, so the change seems a bit weird here...